physics/smoke_dust/rrfs_smoke_wrapper.meta

```diff
@@ -106,6 +106,14 @@
   dimensions = ()
   type = logical
   intent = in
+[sc_factor_in]
+  standard_name = scale_factor_for_wildfire_emissions
+  long_name = scale factor for wildfire emissions
+  units = none
```

Comment:
units should be 1 according to https://github.com/ESCOMP/CCPPStandardNames/blob/main/StandardNamesRules.rst. This change should be here and everywhere else this variable is used in ccpp/physics and fv3atm.